### PR TITLE
Feature/submit shoe

### PR DESCRIPTION
**Pull request by jeremiahblackol**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description
can now submit new shoe

### Why is this change required? What problem does it solve?
interaction with UI and server

### Files modified:
- [ ] SubmitShoeForm.js

### CODE REVIEW

**Comment by Aeickelman40** [comment]
Huuuuge! Nice man, I think all we need is a submission method within this component then the POST method in our apiCalls!

File: src/App/App.js [hunk 1]
```diff
@@ -40,8 +56,13 @@ class App extends Component {
               return <ExpandedShoe {...shoeToRender}/>
             }}
             />
+            <Route 
+              exact path="/submitShoe"
```
Comment (jdvolk) [inline src/App/App.js:60]: looks good!

File: src/SubmitShoeForm/SubmitShoeForm.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import React, {Component} from 'react';
+
+class SubmitShoeForm extends Component {
+    constructor() {
+        super();
+        this.state = {
+            id: 0,
```
Comment (jdvolk) [inline src/SubmitShoeForm/SubmitShoeForm.js:7]: nice use of state

File: src/SubmitShoeForm/SubmitShoeForm.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                placeholder='Price (in dollars)' required>
+            </input>
+            <input
+                name="model"
+                value={this.state.model} 
+                onChange={this.handleChange} 
+                type='text'
+                placeholder='Model' required>
+            </input>
+            <button type="submit" onClick={this.addId}>
```
Comment (jdvolk) [inline src/SubmitShoeForm/SubmitShoeForm.js:82]: form organization looks great

**Pull request merged**